### PR TITLE
feat: add chatwoot

### PR DESCRIPTION
**Pull request by productdevbook**
https://opensource.huntersofbook.com/guide/nuxt-packages/chatwoot

### CODE REVIEW

**Comment by productdevbook** [comment]
@Atinux I changed them all a month ago. When is it confirmed? and https://github.com/nuxt/modules/pull/450 naive ui

File: modules/huntersofbook-chatwoot.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Chatwoot
```
Comment (pi0) [inline modules/huntersofbook-chatwoot.yml:1]:
```suggestion
name: chatwoot
```

File: modules/chatwoot.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+description: >-
+  Chatwoot is an open-source customer engagement 
+  platform that helps companies engage their 
+  ustomers on their website
+repo: huntersofbook/huntersofbook
+npm: '@huntersofbook/chatwoot-nuxt'
+icon: chatwoot.png
+github: https://github.com/huntersofbook/huntersofbook/tree/main/packages/chatwoot-nuxt
+website: https://github.com/huntersofbook/huntersofbook/tree/main/packages/chatwoot-nuxt
+learn_more: ''
```
Comment (pi0) [inline modules/chatwoot.yml:11]:
```suggestion
learn_more: 'https://www.chatwoot.com'
```

File: modules/chatwoot.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+name: chatwoot
+description: >-
+  Chatwoot is an open-source customer engagement 
+  platform that helps companies engage their 
+  ustomers on their website
+repo: huntersofbook/huntersofbook
```
Comment (pi0) [inline modules/chatwoot.yml:6]:
```suggestion
repo: huntersofbook/huntersofbook#main/packages/chatwoot-nuxt
```

**Comment by pi0** [comment]
Thanks

**Pull request merged**